en/10_Contributing/02_Documentation.md

```diff
@@ -93,30 +93,59 @@ yarn lint --fix
 
 ## Writing style
 
-- Write in second person form, addressing the reader. For example "First you'll install a webserver" is good style.
-- If referring to maintainers of Silverstripe CMS, either explicitly say "the maintainers", or write in second person plural form (use "we" instead of "I").
-- Use gender neutral language throughout the document, unless referencing a specific person. Use them, they, their, instead of he and she, his or her.
-- Use simple language and words. Avoid uncommon jargon and overly long words. Remember that not everyone speaks English as their primary language.
-- Use UK English, not US English. Silverstripe CMS is proudly a New Zealand open source project we use the UK spelling and forms of English. The most common of these differences are -ize vs -ise, or -or vs our (eg color vs colour).
-- Write in an active and direct voice.
-- Avoid saying words like "obviously" or "of course". Things that are obvious to you as the writer may not be so obvious to the person reading the documentation.
-- Avoid FAQs. FAQs are not a replacement for coherent, well explained documentation. If you've done a good job
-documenting, there shouldn't be any "frequently asked questions" left.
-- We use sentence case for titles. Only capitalise the first letter of the first word of a title. The only exceptions to this are when using brand names (e.g. GitHub), acronyms (e.g. PHP) and class names (e.g. `ModelAdmin`).
+### Voice and tone
+
+- Use an active voice.
+- Use plain language for easier comprehension.
+- Keep sentences clear and concise.
+
+### Content structure
+
+- Start each page with a clear summary sentence outlining what readers can expect.
+- Structure content around a single topic.
+- Use appropriate markdown. The use of code blocks, lists, tables and callout blocks make technical writing easier to scan.
+- Use descriptive headings and subheadings organised in a hierarchy.
+- Present lists as bullet points.
+- Avoid FAQs as they are not a replacement for coherent, well explained documentation.
+
+### Language and grammar
+
+- Use UK English, adhering to -ize vs -ise, and -or vs -our spelling conventions.
+- Spell out acronyms the first time, with the abbreviation in brackets.
+- Spell out numbers at the beginning of sentences; otherwise, use numerals.
+- Use simple contractions for readability.
+- Only use ampersands for brand or team names.
+- Use gender-neutral language and singular "they" instead of "he/she/his/her".
+- Use "allowlist/denylist" instead of "whitelist/blacklist".
+- Refrain from underlining, italicizing, bolding, or using all caps. Instead use callout blocks where appropriate.
+- Use meaningful link text, i.e. avoid the use of "click here" link text.
+- Use i.e. and e.g. with dots.
+
+### Code blocks
+
+- Use triple backtick code fences to denote code blocks.
+- All code fences must define a language. Use `text` for plain text.
```

Comment:
```suggestion
- All code fences must have an appropriate syntax highlighting hint. Use `text` for plain text.
```
I think this was clearer from the original

Reply:
Done